geomstats/backend/common.py

```diff
@@ -0,0 +1,3 @@
+import numpy as np
```

Comment:
why this new file named common?

Reply:
common variables/computations across frameworks